styles/main.scss

```diff
@@ -829,6 +829,7 @@ h2 {
   width: 102px;
   height: 102px;
   transition: all 0.3s ease-out;
+  position: relative;
```

Comment:
@ntorbinskiy 
Um, does it do anything?

Reply:
Yes, it does. In section Latest posts, Post will fill by his parent WITH position: relative. Without this line- it'll fill the whole screen. That is why I put it in here
![image](https://user-images.githubusercontent.com/60061661/195784426-d888e5b6-4f11-4f0e-94c4-fdb73253d0ac.png)

Reply:
Um, okay, I just thought when you use Next Image it adds automatically